src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -15,6 +15,7 @@ import {
     getVulnerableSeatByVotes,
 } from "../../../utilities/district";
 import { isQuotientAlgorithm } from "../../../computation/logic";
+import { Dictionary } from "utilities/dictionary";
```

Comment:
Codacy found an issue: [Submodule import paths from this package are disallowed; import from the root instead](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)